### PR TITLE
Invoices record endpoints

### PR DESCRIPTION
**Pull request by stovermc**
After reading through the spec again I realized we dont need full crud, only Index and show. In addition we also need to create routes for finding each object by one of its attributes. For example, finding all invoices associated a specific merchant id. This PR does not have the find routes. @themenintights

### CODE REVIEW

File: log/development.log [hunk 1]
```diff
@@ -619,3 +619,15 @@ Migrating to AddTimestampsToMerchants (20170502005406)
   [1m[36mActiveRecord::InternalMetadata Load (0.2ms)[0m  [1m[34mSELECT  "ar_internal_metadata".* FROM "ar_internal_metadata" WHERE "ar_internal_metadata"."key" = $1 LIMIT $2[0m  [["key", :environment], ["LIMIT", 1]]
```
Comment (jmejia) [inline log/development.log:619]: @stovermc Let's add the log directory to the gitignore. We don't want these taking up unnecessary space. I'd recommend adding it to your global gitignore.

File: spec/requests/api/v1/endpoints/invoice_items_requests_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  it "can get one invoice_item id" do
+    id = create(:invoice_item).id
+
+    get "/api/v1/invoice_items/#{id}"
+    invoice_item = JSON.parse(response.body)
+    expect(invoice_item["id"]).to eq id
+  end
+end
```
Comment (jmejia) [inline spec/requests/api/v1/endpoints/invoice_items_requests_spec.rb:21]: @stovermc YAY! TESTS! 🎉

**Review by jmejia — COMMENTED** [review]
Good start @stovermc. I'd recommend making smaller PRs moving forward. Probably could separate Item and InvoiceItem into separate PRs. I do like that they appear to be in separate commits which is nice.

**Comment by seth-at-at** [comment]
This is looking good, nice job noticing we were adding too much functionality.

**Pull request merged**